docs/Developers/Git_Basics.rst

```diff
@@ -110,6 +122,15 @@ It will give you an idea about what files are currently modified.
 Adding the files and commiting
 ------------------------------
 
+First, make sure you're on the correct branch and not developing on master! If
+you've been following this guide, and this is your first pull request,
+you should be developing on the "my-first-good-pull-request" branch.
+You can check your branch with:
+
+::
+
+    $ git status
+
```

Comment:
isn't this a repetition of what is up https://github.com/coala/coala/pull/3729/files#diff-822476541c570d26d3fd9578bd3994b7R107